data/coc.md

```diff
@@ -0,0 +1,43 @@
+### Introduction
+
+DevFest Ukraine is dedicated to providing a harassment-free and inclusive event experience for everyone regardless of gender identity and expression, sexual orientation, disabilities, neurodiversity, physical appearance, body size, ethnicity, nationality, race, age, religion, or other protected category. We do not tolerate harassment of event participants in any form. DevFest Ukraine takes violations of our policy seriously and will respond appropriately.
```

Comment:
Can we replace all the occurrences of `DevFest Ukraine` with `GDG DevFest` or `DevFest Season 2019`?

Reply:
I've updated it to use the same event name as the rest of the site and moved the contacts and form into a json file.